libs/features/webapp-prompt/webapp-prompt.css

```diff
@@ -188,3 +188,119 @@
     }
   }
 }
+
+/* DISMISSAL TOOLTIP */
+
+[data-pep-dismissal-tooltip]::after {
+  content: attr(data-pep-dismissal-tooltip);
+  display: inline-flex;
+  z-index: 3;
+  background-color: #3B63FB;
+  height: fit-content;
+  width: 8.875rem;
+  top: 125%;
+  left: -300%;
+  word-break: break-word;
```

Comment:
I just tested it with Japanese. It seems to be working fine. Are there other languages where this might be an issue?
<img width="293" alt="Screenshot 2024-06-20 at 12 19 12 PM" src="https://github.com/adobecom/milo/assets/118168183/ecf518bb-3bb6-4b0a-80e5-785deb9a1ecd">

Reply:
Visually things are good, yes, but JP has a different system where
In normal Japanese writing, no spaces are left between words, except if the writing is exclusively in [hiragana](https://en.wikipedia.org/wiki/Hiragana) or [katakana](https://en.wikipedia.org/wiki/Katakana) (or with very little [kanji](https://en.wikipedia.org/wiki/Kanji)), in which case spaces may be required to avoid confusion. See https://en.wikipedia.org/wiki/Japanese_punctuation

Reply:
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), `break-word` "Has the same effect as overflow-wrap: anywhere combined with word-break: normal, regardless of the actual value of the [overflow-wrap](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) property.". And `overflow-wrap: anywhere` has the [following description](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap): "To prevent overflow, an otherwise unbreakable string of characters — like a long word or URL — may be broken at any point if there are no otherwise-acceptable break points in the line. No hyphenation character is inserted at the break point. Soft wrap opportunities introduced by the word break are considered when calculating min-content intrinsic sizes."

So I think even in cases where there are no spaces, the line will break when it's necessary (and this is the behavior we want).

Reply:
I'm not arguing with that, I just think we need some validation to make sure things are good. I know we have a special library to deal with word wrap on JP - libs/features/japanese-word-wrap.js but not sure how your code will interact with that.

Reply:
@narcis-radu  This has been verified and working as expected.